webapp/src/components/ImageEditor/ImageCanvas.tsx

```diff
@@ -32,6 +32,8 @@ export interface ImageCanvasProps {
     tilemapState?: TilemapState;
     imageState?: pxt.sprite.ImageState;
     prevFrame?: pxt.sprite.ImageState;
+
+    observeShortcuts: boolean;
```

Comment:
Whoops! Obviously I should invert this prop so that the default behavior is to observer. Fix incoming...